Uppsala/UppEt45.xml

```diff
@@ -0,0 +1,381 @@
+                                መሠርይ፡ ወለሌጌዎን፡ ዕከይ፡ ወለእመ፡ አሐዞ፡ እደ፡ ሰብእ፡ ይትናገር፡ ሕሙም፡ ፌራ፡ በቀቢ፡ ስሙ፡ ለእግዚአብሔር፡ 
+                            </incipit>
+                            <explicit>
+                                እለ<supplied reason="omitted">፡</supplied> ገብሩ፡ እለ፡ ሴዘ፡ እለ፡ ጥልፍልፍ፡ እለ፡ ሌጌዎን፡ 
+                                ይስዓር፡ ሥራያቲክሙ፡ እለ፡ ሀሎክሙ፡ በስዕርተ፡ ር<add place="above">እ</add>ስየ፡ በቅድሜየ፡ 
+                                ወበድሜየ፡ ወበድኅሬየ፡ በየማንየ፡ ወበፀጋምየ፡ ዕቀበኒ፡ ለዓመትከ፡ 
+                                <persName ref="PRS14625WalattaMaryam" role="owner"><hi rend="rubric">ወለተ፡</hi>
+                                    <subst><del rend="effaced" unit="chars" quantity="7"/><add>ማርያም፡</add></subst></persName><hi rend="rubric">እ</hi>
+                            </explicit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i7">
+                            <locus target="#1rc"/>
+                            <title type="complete" ref="LIT7218PPYosomer"></title>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡</hi>
+                                ወመጽኡ፡ እሉ፡ አጋንንት፡ <sic resp="JK">ወእግዝትነ፡</sic> <hi rend="rubric">ማርያምሰ፡</hi>
+                                አእመረት፡ ወ፡ ምክሮሙ፡ ወትቤ፡ ዮሳምር፡ አርሞሳድክኤል፡ አዶናኤል፡ ሮሳ፡ ኪራክስክኤል፡ ርድአኒ፡ 
+                                በዛቲ፡ ሰዓት፡
+                            </incipit>
+                            <explicit>
+                                ወእምዝ፡ ጸለየት፡ በነገረ፡ ዕብራይስጢ፡ ወትቤ<supplied reason="omitted">፡</supplied>
+                                ኪርምያል፡ አፍዳኮዳዮዳ፡ ሕልማና፡ ያሰርስ፡ ብንያክኤል፡ ሶርስሳብ፡ ሮስሳብ፡ እርዳድ፡ 
+                                ለምንያም፡ አልሚስ፡ ጢራጢብንዮስ፡ ስምዓኒ፡ ለአመትከ፡ 
+                                <persName ref="PRS14625WalattaMaryam" role="owner"><hi rend="rubric">ወለተ፡</hi>
+                                <subst><del rend="effaced" unit="chars" quantity="7"/><add>ማርያም፡</add></subst></persName>
+                                ወትቤ፡ እግዝእትነ፡ በነገረ፡ ዕብራይስጢ፡ ሶፎሶሮያሮስቦ፡ ወክርሜል<supplied reason="omitted">፡</supplied>
+                            </explicit>
+                        </msItem>
+                      
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>940</height>
+                                        <width>240</width>
+                                    </dimensions>
+                                    <note>Composed of one strip.</note>
```

Comment:
could you encode the number of strips, see guidelines:   `<note>Consists of <measure unit="strips">3</measure> strips.</note>`